projects/ui/src/hooks/farmer/form-txn/useFarmerFormTxnActions.ts

```diff
@@ -0,0 +1,131 @@
+import { useFormikContext } from 'formik';
+import { useCallback, useMemo } from 'react';
+import { Action } from '@reduxjs/toolkit';
+import { FarmToMode, FarmFromMode } from '@beanstalk/sdk';
+import { FormTxn } from '~/util/FormTxns';
+import useFarmerFormTxnsSummary from './useFarmerFormTxnsSummary';
+import { FormTokenStateNew, FormTxnsFormState } from '~/components/Common/Form';
+import useSdk from '~/hooks/sdk';
+import { ActionType } from '~/util';
+import { ZERO_BN } from '~/constants';
+import useAccount from '~/hooks/ledger/useAccount';
+
+const isClaimingBeansAction = (action: FormTxn) => {
+  const isClaiming =
+    action === FormTxn.CLAIM ||
+    action === FormTxn.HARVEST ||
+    action === FormTxn.RINSE;
+
+  return isClaiming;
```

Comment:
What do you think about just returning this?
```

return  action === FormTxn.CLAIM ||
    action === FormTxn.HARVEST ||
    action === FormTxn.RINSE;
```

Or  use the arrow function without the curly braces